setup.py

```diff
@@ -17,121 +17,119 @@
-          'Programming Language :: Python :: Implementation :: CPython',
-          'Programming Language :: Python :: Implementation :: PyPy',
-          'Natural Language :: English',
-          'Operating System :: OS Independent',
-          'Topic :: Internet :: WWW/HTTP',
-          'Framework :: Zope :: 3',
-      ],
-      url='http://github.com/zopefoundation/zc.catalog',
-      license='ZPL 2.1',
-      packages=find_packages('src'),
-      package_dir={'': 'src'},
-      namespace_packages=['zc'],
-      extras_require={
-          'test': tests_require,
-          'browser': [
-              'zope.app.form',
-              'zope.browsermenu',
-          ],
-          'stemmer': stemmer_requires,
-      },
-      tests_require=tests_require,
-      install_requires=[
-          'BTrees >= 4.4.1',
-          'persistent',
-          'pytz',
-          'setuptools',
-          'zope.catalog >= 4.2.1',
-          'zope.component >= 4.3.0',
-          'zope.container >= 4.1.0',
-          'zope.i18nmessageid >= 4.1.0',
-          'zope.index >= 4.3.0',
-          'zope.interface >= 4.4.0',
-          'zope.intid >= 4.2.0',
-          'zope.publisher >= 3.12',
-          'zope.schema >= 4.4.2',
-          'zope.security >= 4.1.0',
-      ],
-      include_package_data=True,
-      zip_safe=False,
+setup(
+    name="zc.catalog",
+    version="2.0.2.dev0",
+    author="Zope Corporation and Contributors",
+    author_email="zope-dev@zope.org",
+    description="Extensions to the Zope 3 Catalog",
+    long_description=(
+        read("README.rst")
+        + "\n\n.. contents::\n\n"
+        + read("CHANGES.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "valueindex.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "setindex.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "normalizedindex.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "extentcatalog.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "stemmer.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "legacy.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "globber.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "callablewrapper.rst")
+        + "\n\n"
+        + read("src", "zc", "catalog", "browser", "README.rst")
+    ),
+    keywords="zope3 i18n date time duration catalog index",
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Environment :: Web Environment",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Zope Public License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
```

Comment:
I did not review, just noticed that 3.7 was not added here in the list of classifiers, this seems to be a mistake.